frontend/13f-app/src/components/filingdata.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import axios from 'axios'
+import './filingdata.css'
+
+function FilingData () {
+  const [results, setResults] = useState([])
+  const { filingId } = useParams()
+  useEffect(() => {
+    axios.get(`${process.env.REACT_APP_API_SERVER}/edgarfiling/${filingId}/data/`)
+      .then(res => {
+        const data = res.data
+        setResults(data)
+      })
+    // axios.get(`${process.env.REACT_APP_API_SERVER}/company/${companyId}`)
```

Comment:
Rose's stuff is in so you should be able to uncomment this now

Reply:
Completed